### ISSUE TITLE
Add toggle to grey out / disable panel in Side Navigation Component

### ISSUE THREAD

**Issue by bserrano1** [GSA/sam-design-system#1529]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is
We are currently asked to disable / grey out the R/Q panel when a user is viewing a historical entity. 

**Describe the desired solution**
We are looking to see if there can be an added property to the NavigationLinks object that can allow disabling per panel as well as the possibility of hover over text to explain why a user cannot click on the panel.

**Describe alternatives you've considered**
N/A

**Additional context**
![image](https://github.com/user-attachments/assets/21f1377b-37dc-4f4c-a73b-f8c9fa922cc4)

**Comment by cwolf10** [comment]
Hey @bserrano1, just to clarify what you're looking for, would this feature be to disable individual links or just to disable the whole panel?

Also, is there a requirement at this point to display a tooltip when disabled? Or are you just anticipating that it might be a desirable feature in the future?

**Comment by bserrano1** [comment]
Hello @cwolf10, 
"would this feature be to disable individual links or just to disable the whole panel?"

- It would be to disable the whole panel. For example, the Responsibility / Qualification panel would be greyed out and the user would not be able to interact / click it for routing.

Also, is there a requirement at this point to display a tooltip when disabled? Or are you just anticipating that it might be a desirable feature in the future?

- Yes, there is a requirement to add a tooltip along with the disabled panel to explain to the user why they are unable to click on that specific panel.

**Comment by cwolf10** [comment]
@bserrano1 Last thing, do you have mockups of how the side nav should appear when disabled? Also is there one of the tooltip showing on the disabled sidenav?

**Comment by bserrano1** [comment]
At this time, we have not been provided with any mockups.

**Comment by shannoncates-gsa** [comment]
@bserrano1 @cwolf10 the design team is waiting on the resolution of this ticket to move forward with mockups. We have never employed a tooltip on a grayed out nav-link before so we'd like to know the feasibility of it before we work on a new styling.

**Comment by cwolf10** [comment]
@shannoncates-gsa I don't see why this would not feasible. Hopefully that allows design to move forward with mockups.

As far as the resolution of this ticket, we can't really move forward on development without the mockups/Figma showing what the updated look and behavior should be.

**Comment by cwolf10** [comment]
@bserrano1 I am going to close this issue for now since there isn't any work for the design team to do at this point. Once there are mockups of how the sidenav should look when disabled and how the tooltip should look on the disabled sidenav feel free to re-open this issue and we can see about getting those updates made.

**Comment by Sunset05** [comment]
We have recently received updated mocks for how this should look. To re-open this issue do i need to create a new issue? or can we use this same ticket?

**Comment by Sunset05** [comment]
Okay thanks, attaching below!

This mock shows the Responsibility/Qualification tab grayed and disabled.

![image](https://github.com/user-attachments/assets/55f1c46a-8c02-48e4-aa0c-43f3dda08fee)

**Comment by cwolf10** [comment]
Do you have a mockup that shows the tooltip when hovering over it? Or is that still a requrement?

**Comment by shannoncates-gsa** [comment]
The tooltip is no longer needed.

*Shannon Cates*
*UX Designer - Design Systems Team*
Contractor | IBM
she/her

**Comment by bserrano1** [comment]
@cwolf10 just wanted to see if there was anything else that was needed before the work can begin.
